Python/Tests/ProjectUITests/LinkedFileTests.cs

```diff
@@ -665,10 +666,11 @@ public class LinkedFileTests {
 
                         // verify that the project file contains the correct text for Link
                         var fileText = File.ReadAllText(project.FullName);
-                        var pattern = string.Format(
+                        /*var pattern = string.Format(
                             @"<Content Include=""{0}"">\s*<Link>{1}</Link>\s*</Content>",
                             Regex.Escape(tempFile),
-                            Regex.Escape(Path.GetFileName(tempFile)));
+                            Regex.Escape(Path.GetFileName(tempFile)));*/
+                        var pattern = PathUtils.GetRelativeFilePath(Path.GetDirectoryName(tempFile), tempFile);
```

Comment:
I seem to have misunderstood what you mean. I think it's just a matter of matching the values obtained by "PathUtils. GetRelative FilePath" directly. Actually, it doesn't seem to be able to do that.            
As for the NoClickable Point Exception exception, I also had this problem during debugging which led to the failure of debug. This problem also exists in other cases. I don't know how to solve this exception, so I can only see the value of the variable by adding code printing.
I made new changes to the code as follows:
![image](https://user-images.githubusercontent.com/39871640/64945429-1380fc80-d8a3-11e9-91d8-6e8eacd93798.png)
The red box is the part I added. I don't know if I'm right to use the "PathUtils. GetRelative FilePath" method because I can't debug. According to the code I wrote,the value of  "patternTemp" is **<ig1hfmvf.f0t\\\xakgirwu.dum>** and the data we want to match in fileText is <Content="..\..\ig1hfmvf.f0t\xakgirwu.dum"> The problem is that the value of "patternTemp" is not in the format "..\..\".   so I did a patternPath, and I don't know if you have any better suggestions.